<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Structures modeling tool</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>591</width>
      <height>430</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> Synapses:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string> Actions log:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string> Hamiltonian:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string> Distribution of recognized sequences:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew_structure">
     <property name="title">
      <string>New structure</string>
     </property>
     <addaction name="action2D"/>
     <addaction name="action3D"/>
    </widget>
    <addaction name="menuNew_structure"/>
    <addaction name="actionEdit_stucture"/>
    <addaction name="actionSave_structure"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_QT"/>
   </widget>
   <widget class="QMenu" name="menuNetwork">
    <property name="title">
     <string>Network</string>
    </property>
    <widget class="QMenu" name="menuRun_test">
     <property name="title">
      <string>Run test</string>
     </property>
     <addaction name="actionFast_test"/>
     <addaction name="actionOrdinary_test"/>
    </widget>
    <addaction name="actionCreate_net_from_structure"/>
    <addaction name="actionDelete_net"/>
    <addaction name="actionSave_net_to_file"/>
    <addaction name="actionOpen_net"/>
    <addaction name="separator"/>
    <addaction name="menuRun_test"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCreate_inputs"/>
    <addaction name="actionSave_distribution"/>
    <addaction name="actionSave_Hamiltonian"/>
    <addaction name="actionUpdate_diagramm"/>
    <addaction name="actionShow_distribution_info"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNetwork"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSave_structure">
   <property name="text">
    <string>Save structure</string>
   </property>
   <property name="statusTip">
    <string>Saves file with sinapses</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About program</string>
   </property>
  </action>
  <action name="actionAbout_QT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="actionEdit_stucture">
   <property name="text">
    <string>Edit stucture</string>
   </property>
   <property name="statusTip">
    <string>Open structure editor</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionCreate_net_from_structure">
   <property name="text">
    <string>Create net from structure</string>
   </property>
   <property name="statusTip">
    <string>Creates neural net from structure</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave_net_to_file">
   <property name="text">
    <string>Save net to file</string>
   </property>
   <property name="statusTip">
    <string>Saves net to file</string>
   </property>
  </action>
  <action name="actionDelete_net">
   <property name="text">
    <string>Delete net</string>
   </property>
   <property name="statusTip">
    <string>Deletes neural net from memory</string>
   </property>
  </action>
  <action name="actionOpen_net">
   <property name="text">
    <string>Open net</string>
   </property>
   <property name="statusTip">
    <string>Opens neural net from file</string>
   </property>
  </action>
  <action name="actionCreate_inputs">
   <property name="text">
    <string>Create inputs</string>
   </property>
   <property name="statusTip">
    <string>Creates file with inputs for testing</string>
   </property>
  </action>
  <action name="actionFast_test">
   <property name="text">
    <string>Fast test</string>
   </property>
   <property name="statusTip">
    <string>Runs test with storing data in DB</string>
   </property>
  </action>
  <action name="actionOrdinary_test">
   <property name="text">
    <string>Ordinary test</string>
   </property>
   <property name="statusTip">
    <string>Runs test with storing data in memory (for low number of neurons)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave_distribution">
   <property name="text">
    <string>Save distribution</string>
   </property>
   <property name="statusTip">
    <string>Saves distribution diagramm to file</string>
   </property>
  </action>
  <action name="actionUpdate_diagramm">
   <property name="text">
    <string>Update diagramm</string>
   </property>
  </action>
  <action name="actionShow_distribution_info">
   <property name="text">
    <string>Show distribution info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSave_Hamiltonian">
   <property name="text">
    <string>Save Hamiltonian</string>
   </property>
  </action>
  <action name="action2D">
   <property name="text">
    <string>2D</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="action3D">
   <property name="text">
    <string>3D</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionTest_with_given_inputs">
   <property name="text">
    <string>Test with given inputs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
